tubemq-server/src/main/java/org/apache/tubemq/server/broker/metadata/BrokerDefMetadata.java

```diff
@@ -32,7 +32,7 @@
     private int numPartitions = 1;
     // data will be flushed to disk when unflushed message count exceed this.
     private int unflushThreshold = 1000;
-    @Deprecated
+    // data will be flushed to disk when unflushed data size reaches the threshold.
     private int unflushDataHold = 10000;
```

Comment:
The initial value of unflushdatahold is set to 0  to turn off this feature by default.